client/styles/typography.ts

```diff
@@ -57,7 +58,7 @@ export const PageTitle = styled.h1`
   font-size: ${desktopFontSize.h1};
   line-height: 120%;
   color: ${colours.black};
-  @media ${device.laptop} {
+  @media ${device.tablet} {
```

Comment:
It was decided in the design review. We gradually made the page narrower, and the title shrank sooner than we expected. This change makes the breakpoint happen on mobile screen sizes only, rather than mobile and tablet